docs/DeveloperGuide.adoc

```diff
@@ -292,23 +292,75 @@ Refer to the guide <<DevOps#, here>>.
+
+|`* * *` |diabetic |reminded to take my insulin|  regularly
+
+|`* * *` |diabetic |be able to track my sugar levels| 
+
+|`* * *` |task-oriented diabetic patient |have a goal to work towards or a challenge to work on everyday |have a sense of direction in what I can do to improve my health
+
+|`* *` |caretaker of an elderly patient with diabetes whose family members are busy working |reliable app to keep track of all the patients' activities |can answer to the family members who have entrusted unto me this responsibility of care
+
+|`* *` |busy person |be able to easily sort and prioritize my tasks |better manage my time
+
+|`* *` |diabetic patient who is often being referred to new doctors at different specialist clinics every now and then |be able to be able to export all my records and activities at once |rule out the possibility of missing any information during the registration process at a new clinic/ hospital I am visiting
+
+|`* *` |family member of a diabetic |prioritize my tasks |be immediately contactable if my family member has an emergency situation that requires urgent medical attention
+
+|`* *` |diabetic |have a customisable app with avatars and different backgrounds |enjoy a personalised experience
+
+|`* *` |lazy and obese individual |be motivated constantly to exercise |stop procrastinating
+
+|`* *` |forgetful diabetics patient |have a record of my doctors' advice for each medical appointment and prescription directions |better understand the steps that I can take to improve my condition until the next consultation
+
+|`* *` |achievement-oriented diabetic | view the achievements and progress I have made on food intake |remain motivated to keep my streak on good habits going
+
+|`* *` |paranoid diabetic who values privacy |secure/encrypt my health data and other <<private-contact-detail,private contact details>> |protect my data
+
+|`* *` |diabetic patient with a family |have a user-friendly app that helps me manage my medical data and appointments on my own |free the burden I have on my family
+
+|`* *` |diabetic patient with a family |have a user-friendly app with natural commands that helps me manage my medical data and appointments on my own |free the burden I have on my family
+
+|`*` |diabetic patient in a community of diabetic patients |have a standardised means of comparing our activities via a social network | learn from my peers, encourage and be encouraged through this difficult journey.
+
+|`*` |careless user  |undo my most recent actions |easily make necessary amendments and input the correct commands
+
+|`*` |a diabetic patient who has many medical receipts - and is not very good at mathematics |have a simple calculator that is always easily accessible |instantly calculate all my medical costs when needed
+
+|`*` |an obese working adult at high risk of diabetes |start monitoring my diet |minimise my risk of having diabetes
+
+|`*` |medical consultant |export my patient's health data  |save my time
```

Comment:
Do you intend to target medical consultant as well? If so, you might want to consider providing more user stories for them